analyzer/runtime/extract.go

```diff
@@ -523,7 +524,7 @@ func sumGasUsed(events []*EventData) (sum uint64, foundGasUsedEvent bool) {
 	return
 }
 
-func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Address]bool, eventsRaw []nodeapi.RuntimeEvent) ([]*EventData, error) {
+func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Address]bool, eventsRaw []nodeapi.RuntimeEvent) ([]*EventData, error) { //nolint:gocyclo
```

Comment:
weird that it thinks the complexity of extractEvents has increased, there's only entries added to a map literal :upside_down_face:

Reply:
the extract function must grow. marking as resolved